models/base.py

```diff
@@ -322,10 +322,10 @@ def safe_mkdir(cls, path):
     )
 
     format = sa.Column(
-        im_format_enum,
+        file_format_enum,
         nullable=False,
         default='fits',
-        doc="Format of the file on disk. Should be fits or hdf5. "
+        doc="Format of the file on disk. Should be fits, hdf5, csv or npy. "
```

Comment:
Don't act on this now, but : some year we might also want to support Parquet files.  (The library pyarrow handles those.  I came across these in another project.)

I'm not sure that parquet gives us anything we don't already have with hdf5, though.  I think the interface might be simpler for reading a smaller dataset (e.g. a table of objects from a single image).